ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,410 @@
+      <FlexColumn style={{justifyContent: 'space-evenly', width: '100%'}}>
+        <div style={styles.textHeader}>What payment method would you like to use?</div>
+        <FlexColumn style={{marginTop: '20px', width: '100%'}}>
+          <FlexRow style={{boxSizing: 'border-box', borderRadius: '8px', border: '1px solid #CCCFD4', marginBottom: '7px'}}>
+            <RadioButton data-test-id='credit-card-radio'
+                         style={{margin: '15px', height: '17px', width: '17px'}}
+                         checked={useCreditCard === true}
+                         onChange={() => setUseCreditCard(true)}/>
+            <FlexColumn style={{marginTop: '9px', marginLeft: '15px', marginBottom: '15px'}}>
+              <FlexRow>
+                <div style={styles.textHeader}>Credit Card&nbsp;&nbsp;</div>
+                <i style={styles.textNormal}>24 hours to process</i>
+              </FlexRow>
+              <div style={styles.textNormal}>A Google billing partner representative will contact you to process
+                your request.</div>
+            </FlexColumn>
+          </FlexRow>
+          <FlexRow style={{boxSizing: 'border-box', borderRadius: '8px', border: '1px solid #CCCFD4', marginBottom: '7px'}}>
+            <RadioButton
+                         style={{margin: '15px', height: '1.063rem', width: '1.063rem'}}
+                         checked={useCreditCard === false}
+                         onChange={() => setUseCreditCard(false)}/>
+            <FlexColumn style={{marginTop: '9px', marginLeft: '15px', marginBottom: '15px'}}>
+              <FlexRow>
+                <div style={styles.textHeader}>Purchase Order/Other&nbsp;&nbsp;</div>
+                <i style={styles.textNormal}>5-7 days to process</i>
+              </FlexRow>
+              <div style={styles.textNormal}>You will need to provide more info for the quote, a
+                Google billing partner representative will contact you to process your request.
+              </div>
+            </FlexColumn>
+          </FlexRow>
+          <FlexRow style={{marginTop: '20px'}}>
+            <CheckBox style={{margin: '15px', height: '1.063rem', width: '1.063rem'}}
+                      checked={nihFunded === true}
+                         onChange={() => setNihFunded(false)}/>
+            <FlexColumn style={{marginTop: '9px', marginLeft: '15px', marginBottom: '15px'}}>
+                <div style={styles.textHeader}>NIH-funded Research.</div>
+              <div style={styles.textNormal}>My research is funded by the National Institute of Health, (NIH).
+                NIH funded research is eligible for discounted cloud rates through
+                the NIH STRIDES initiative.
+              </div>
+            </FlexColumn>
+          </FlexRow>
+        </FlexColumn>
+        <FlexRow style={{marginTop: '100px', justifyContent: 'space-between'}}>
```

Comment:
These 3 buttons feel like they can also be pulled out and generalized, outside of the `switchCase`

Reply:
I failed to do so because they are slightly different.
Like 1st step no Back button, the others have, so I have to create one more layer of Flex for the other steps.
And the final step is Submit with a different function. Extracting common method end up having if(step === 1), else if(step ===2)